package.json

```diff
@@ -71,7 +71,8 @@
     "source-map-support": "0.5.19",
     "typescript": "3.9.7",
     "webpack": "5.36.2",
-    "webpack-cli": "4.4.0"
+    "webpack-cli": "4.4.0",
+    "proxy-agent": "latest"
```

Comment:
1. is `proxy-agent` needed as a dev-dep in multiple sub-packages or only in one?
If it is **not** in multiples it should be a dev-dep of the specific package.

2. Why do we depend on `latest` version?

Reply:
deleted from here.